src/Rings/mpoly-localizations/methods.jl

```diff
@@ -0,0 +1,28 @@
+
+@attr Vector{<:MPolyLocRingElem} function minimal_generating_set(
+    I::MPolyLocalizedIdeal{<:MPolyLocRing{<:Field, <:FieldElem, 
+                                          <:MPolyRing, <:MPolyElem, 
+                                          <:MPolyComplementOfKPointIdeal}
+                          }
+  )
+  L = base_ring(I)::MPolyLocRing # Type annotation for the reader's convenience.
+  R = base_ring(L)::MPolyRing
+
+  shift, back_shift = base_ring_shifts(L) # The shifting maps of R moving the localization point to 
+                                          # origin and back. 
+  I_shift = shifted_ideal(I)::MPolyIdeal # The pre_saturated_ideal of I in R after the shift 
+                                         # moving the localization point to the origin
+  o = negdegrevlex(gens(R)) # the default local ordering
+
+  std_I_shift = standard_basis(I_shift, ordering=o)::IdealGens # Refers to a pre-computed standard basis if there is any.
+  bpa = std_I_shift.gens::BiPolyArray # Only the BiPolyArray has a "singular side".
+  l_sing = Singular.minimal_generating_set(bpa.S) # This should have the correct ordering set due 
+                                                  # to the construction history. TODO: Please confirm!
```

Comment:
@jankoboehm @ederc

Reply:
I am neither @jankoboehm nor @ederc , but from my limited understanding, we do not want to see an explicit BiPolyArray at this level of code. Paralleling the implementation of minimal_generating_set for graded rings, I would rather expect lines 17-22 to be something like
`
  singular_assure(I_shift,o)
  I_shift_min = Singular.minimal_generating_set(I_shift.gens.gens.S)
  I_min = L.(back_shift.(I_shift_min))
`
(of course fixing types whereever necessary....)

Please note that the side-effect of mstd computing a standard basis, which is exploited in the graded case, does not exist for minbase in the local case in Singular. So there might be no point in spending computation time on the standard basis here.